backend/editor/entries.py

```diff
@@ -10,8 +10,8 @@ class TaxonomyGraph:
     """Class for database operations"""
     
     def __init__(self, branch_name, taxonomy_name):
-        self.multi_label = ('t_'+taxonomy_name)+':'+('b_'+branch_name)
-        self.multi_label_search = ('t_'+taxonomy_name)+'_'+('b_'+branch_name)
+        self.project_name = 'p_' + taxonomy_name + '_' + branch_name
+        self.multi_label = self.project_name + ':' + ("t_" + taxonomy_name) + ":" + ("b_" + branch_name)

```

Comment:
you only need the project name here, as this is redundant with the other labels.

We keep the other label just for future work (or being able to explore the database independently).